dch/public/app/components/watch-form/watch-form.html

```diff
@@ -0,0 +1,37 @@
+<div class="container" style='align:left'>
+
+<form name = "myWatchForm" ng-submit="submitForm(selectedWatch.id, selectedWatch.startTime, selectedWatch.interval)">
+	<label for="watchSelect">Select a watch ID:</label>
+
+ 	<select name="watchSelect" ng-model="selectedWatch.id">
```

Comment:
There are a few deviations from the John Papa style guide in this directive. Ideally, you want to access items in the controller in dot notation rather than relying on scopes.

E.g. 

```
ng-model="selectedWatch.id"
```

becomes

```
ng-model="WatchFormController.selectedWatch.id"
```

This doesn't work if ```selectedWatch``` is attached to a scope but you can find more details below.

See:

https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#controllers